src/NServiceBus.Core/ObjectBuilder/IBuilder.cs

```diff
@@ -45,19 +45,5 @@ public interface IBuilder : IDisposable
         /// <param name="typeToBuild">The <see cref="Type" /> to build.</param>
         /// <returns>The component instances.</returns>
         IEnumerable<object> BuildAll(Type typeToBuild);
-
-        /// <summary>
-        /// Releases a component instance.
-        /// </summary>
-        /// <param name="instance">The component instance to release.</param>
-        void Release(object instance);
```

Comment:
Not sure if it is safe to remove release. This needs to be verified. I thought in Windsor you need to release stuff you acquired.